Gruntfile.js

```diff
@@ -1,14 +1,18 @@
+
 module.exports = function(grunt) {
   'use strict';
 
   // Just set shell commands for running different types of tests
   grunt.initConfig({
-    _test_runner: 'turbo',
-    _unit_args: 'test/test_reporting_client.js test/test_mbaas_reporting.js test/test_sync.js',
+    _test_runner: 'mocha',
+    _unit_args: '-A -u exports --recursive -t 10000 ./test/unit',
 
     // These are the properties that grunt-fh-build will use
-    unit: '<%= _test_runner %> <%= _unit_args %>',
-    unit_cover: 'istanbul cover --dir cov-unit <%= _test_runner %> -- <%= _unit_args %>'
+    unit: '<%= _test_runner %> <%= _unit_args %> mocha --exit',
```

Comment:
Is `mocha --exit` necessary here? Is it specified correctly, or should it just be `--exit`? The 'mocha' part should be covered by `<%= _test_runner %>`, right?

How come it's needed here, and not in `unit_cover` on the next line, is that because `istanbul` manages the process better?

Reply:
Yes. The exit is required in the mocha, probably because of the way that the test are implemented. After all be processed they get stuck so it required the exit.

Reply:
How about the rest of my questions? :P 

* Should it just be `--exit` instead of `mocha --exit`, since `mocha` is there at the start of the command?
* Is it necessary for the `unit_cover` command also?

Also, could you try `_mocha` instead of `mocha`, and without `--exit`...does that give the same result? That's the pattern we've used in other modules up to now.

Reply:
I tried it too. Did not work. The same issue is faced.

Reply:
ok :+1: